src/usr/local/www/interfaces_lagg_edit.php

```diff
@@ -70,56 +70,56 @@
 
 $portlist = get_interface_list();
 $laggprotos	  = array("none", "lacp", "failover", "fec", "loadbalance", "roundrobin");
-$laggprotosuc = array("NONE", "LACP", "FAILOVER", "FEC", "LOADBALANCE", "ROUNDROBIN");
+$laggprotosuc = array(gettext("NONE"), gettext("LACP"), gettext("FAILOVER"), gettext("FEC"), gettext("LOADBALANCE"), gettext("ROUNDROBIN"));
```

Comment:
Note: I did the getttext() here and then used $laggprotosuc[] in generating $protohelp so that
1) The values displayed to the user in the dropdown list can be localized if somebody chooses to in some other language.
2) The help text will follow whatever values are displayed to the the user in the dropdown list.